src/ngraph/op/util/attr_types.hpp

```diff
@@ -49,5 +51,33 @@ namespace ngraph
             AUTO = SAME_UPPER,
             NOTSET = EXPLICIT
         };
+
+        enum class AutoBroadcastType
+        {
+            NONE = 0,
+            NUMPY
+        };
+
+        struct AutoBroadcastSpec
+        {
+            AutoBroadcastSpec()
+                : type(AutoBroadcastType::NONE)
+                , axis(0)
+            {
+            }
+            AutoBroadcastSpec(AutoBroadcastType t)
+                : type(t)
+                , axis(0)
+            {
+            }
+            AutoBroadcastSpec(AutoBroadcastType t, size_t s)
+                : type(t)
+                , axis(s)
+            {
+            }
+
+            AutoBroadcastType type;
+            size_t axis;
```

Comment:
What is axis ? Couldn't find a use anywhere.

Reply:
Used to indicate the start of alignment, I added a comment.